index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="one">Hello Microverse!</header>
```

Comment:
[Optional]
- This is a good use of semantic HTML, and it'd be even better if you wrap the "Hello Microverse!" text inside a <hx> tag.
Usually the <header> tag is used as a container for the whole header of the page which is the <nav> tag.